CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Document textarea ID option ([PR #1410](https://github.com/alphagov/govuk_publishing_components/pull/1410))
```

Comment:
It doesn't affect the component API, so I'm not convinced about the value of this 🤷‍♂

Reply:
The value of adding it to the changelog, or the value of the entire PR?

Reply:
The changelog entry, sorry for the ambiguity. All features should be documented, exemplified and tested 🙌

Reply:
Ah cool. I'll take it out of the changelog then 👍